tests/testthat/test-queuefallout.R

```diff
@@ -11,12 +11,11 @@ test_that("Validate QueueFallout using legacy credentials", {
                      metric="pageviews",
                      element="page",
                      c("http://randyzwitch.com/r-amazon-ec2", "http://randyzwitch.com/julia-import-data")
```

Comment:
Apologies, I was not using a good txt editor and not actually running the tests so I couldn't see the areas.  Using RStudio, I corrected the red x's in the Test-QueueFallout file and I checked the other test-queues for red x's and didn't see any.  Pushing update now.